### ISSUE TITLE
Empty error on update to alpha5 

### ISSUE THREAD

**Issue by Giorgi625** [JoomCoder-com/JoomCCK#67]
I get this when installing alpha 5 over alpha 4, Joomla 4.2.9
![update-error](https://user-images.githubusercontent.com/16633699/235350577-6210a912-b838-4073-81f9-ceeebb5bb1a1.png)

https://cck.joomla.ge on this site. your account which I created before has super user rights so you can check it.

**Comment by andiamine** [comment]
i was able to update :)

**Comment by Giorgi625** [comment]
On my site? I tried again and got same error.
And another. I was not able to update on another website as well. On that site I just had installed JoomCCK alpha 4 and have not done anything else and now wanted to update it, but it did not worked. Then I uninstalled it fully and installed new Joomla 4.3.0 and tried to install JoomCCK but got this error:
![Can not clean install](https://user-images.githubusercontent.com/16633699/235362426-5ced4ef0-dfbc-4cf3-b571-03f479612ead.png)
